api/db/seeds/data/team-acces/build-sco-organizations.js

```diff
@@ -86,6 +92,28 @@ function _buildJosephineBaker(databaseBuilder) {
   ].forEach(_buildAdminMembership(databaseBuilder));
 }
 
+function _buildScoManagingStudentsNonGarOrganization(databaseBuilder) {
```

Comment:
Choisir entre `NonGar` et `NoGar` pour avoir un nommage cohérent partout : 

```suggestion
function _buildScoManagingStudentsNoGarOrganization(databaseBuilder) {
```